packages/core/lib/index.js

```diff
@@ -99,9 +99,13 @@ const evalExpr = async (expr, scope) => {
  * @returns JS return value from the rule. Can be a promise
  */
 const evalRule = async (rule, scope) => {
-  assert(rule.type === "Rule");
   const { fn, args } = rule;
-  assert(fn.type === "Word");
+
+  const resolvedOpts = {};
+  for (const opt of fn.options) {
+    assert(resolvedOpts[opt.key.name] === undefined); // No duplicate option keys
```

Comment:
Assert this with a message, or maybe with a descriptive error for debugging.

Reply:
Yeah, I should actually make this a `BrowseError`. Fixing

Reply:
![image](https://user-images.githubusercontent.com/1797812/90698948-e23d4900-e236-11ea-8686-ce3139dc5cc9.png)